src/routes/+page.server.ts

```diff
@@ -1,5 +1,3 @@
 import { validateSession } from '$lib/server/auth';
 
-export async function load(event) {
-	await validateSession(event);
-}
+export const load = async (event) => validateSession(event);
```

Comment:
at this point you could just do `export const load = validateSession;`

Reply:
for the sake of consistency and a bit readability, I will keep it like this